### PR TITLE
New package: toxcore-git-20150309_1

### PR DESCRIPTION
**Pull request by teh-jazzman**
Library for the TOX instant messenger protocol, used by several clients. I will create pull requests for the clients after this and filteraudio are through.

### CODE REVIEW

**Comment by radare** [comment]
Awesome! Thanks!

File: srcpkgs/toxcore-git/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+do_fetch() {
+	git clone https://github.com/irungentoo/toxcore.git
+}
+
+pre_configure() {
+	./autogen.sh
+}
+
+# Development package
+toxcore-git-devel_package() {
```
Comment (Gottox) [inline srcpkgs/toxcore-git/template]: please rename the subpackage to toxcore-devel-git.

**Comment by Gottox** [comment]
looks good so far :+1:

File: srcpkgs/toxcore-git/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+maintainer="Stefan Mühlinghaus <jazzman@alphabreed.com>"
+license="GPL-3"
+homepage="https://tox.im"
+makedepends="libsodium-devel opus-devel libvpx-devel"
+hostmakedepends="git autoconf automake libtool pkg-config"
+build_style="gnu-configure"
+wrksrc="toxcore"
+
+do_fetch() {
+	git clone https://github.com/irungentoo/toxcore.git
```
Comment (xtraeme) [inline srcpkgs/toxcore-git/template]: Ditto for this one.

**Comment by Gottox** [comment]
please solve the merge conflicts, will merge afterwards

**Pull request merged**